aaelf64/aaelf64.rst

```diff
@@ -1582,6 +1582,10 @@ Unallocated relocations
 
 All unallocated relocation types are reserved for use by future revisions of this specification.
 
+PAuthABI relocations
+^^^^^^^^^^^^^^^^^^^^
+
+The PAuthABI relocations are currently defined in the vendor experiment space. Arm reserves codes 580 to 600 for static PAuthABI relocations and 1040 - 1060 for dynamic PAuthABI relocations.
```

Comment:
I've got no issues with the reserved ranges. But perhaps reword the paragraph slightly to indicate that the plan is to migrate the relocations currently defined in the vendor experiment stage to the reserved ranges. As it's phrased now it's not totally clear to me how the two sentences relate to each other.